pkg/utils/helper.go

```diff
@@ -204,3 +209,20 @@ func IsNitroInstance(instanceType string) (bool, error) {
 	}
 	return false, nil
 }
+
+// GetSourceAcctAndArn constructs source acct and arn and return them for use
+func GetSourceAcctAndArn(roleARN, region, clusterName string) (string, string, error) {
+	// ARN format (https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html)
+	// arn:partition:service:region:account-id:resource-type/resource-id
+	// IAM format, region is always blank
+	// arn:aws:iam::account:role/role-name-with-path
+	roleParts := strings.Split(roleARN, ":")
```

Comment:
let's use https://docs.aws.amazon.com/sdk-for-go/api/aws/arn/ pkg to deal with arn instead of split ourself.

Reply:
Updated to use sdk API.